witchcraft-server/src/logging/mod.rs

```diff
@@ -59,10 +67,50 @@ pub(crate) async fn init(
     let request_logger = logger::appender(install, metrics, hooks).await?;
     let audit_logger = logger::appender(install, metrics, hooks).await?;
 
+    let audit_logger = Arc::new(Mutex::new(audit_logger));
+
+    AUDIT_LOGGER
+        .fill(audit_logger.clone())
+        .map_err(|_| Error::internal_safe("Audit logger already initialized"))
```

Comment:
No need to map the error if you're just going to unwrap

Reply:
I think we need to map to an error because the `Err` case for `fill` just returns the argument, which doesn't implement `Debug`

Reply:
Ah right, then in that case I'd do `fill().ok().expect("audit logger already initialized")`

Reply:
Ah! That is much cleaner. Okay, updated!